main.py

```
@@ -538,9 +658,40 @@ def receive_drawing_job(
+            outputs=[*output_combo, drawing_generate_btn, drawing_continue_btn],
             queue=False,
         )
+        drawing_refresh_models_btn.click(
+            refresh_drawing_models,
+            inputs=[cookies],
+            outputs=[drawing_model, drawing_model_status],
+            queue=False,
+        )
+        def on_drawing_model_changed(model):
+            model_text = model or "未选择"
+            return gr.update(label="绘画区-模型：" + model_text)
```

Ruff (0.16.2)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)